React/Views/RefreshControl/RCTRefreshControlManager.m

```diff
@@ -25,6 +25,7 @@ - (UIView *)view
 RCT_EXPORT_VIEW_PROPERTY(tintColor, UIColor)
 RCT_EXPORT_VIEW_PROPERTY(title, NSString)
 RCT_EXPORT_VIEW_PROPERTY(titleColor, UIColor)
+RCT_EXPORT_VIEW_PROPERTY(progressViewOffset, float)
```

Comment:
Let's make this into `CGFloat`. It better expresses the use of the property.

Reply:
Updated, thanks for the feedback!